src/overlays/actors/ovl_Demo_Gt/z_demo_gt.c

```diff
@@ -1117,7 +1130,7 @@ void func_80980BFC(DemoGt* this, PlayState* play) {
     Vec3f pos;
     u16 csCurFrame = play->csCtx.curFrame;
 
-    if ((csCurFrame == 740) || (kREG(4) == 1)) {
+    if ((csCurFrame == 740) || (OOT_DEBUG && kREG(4) == 1)) {
```

Comment:
```suggestion
    if ((csCurFrame == 740) || (OOT_DEBUG && (kREG(4) == 1))) {
```
Officially not required (and not my personal preference) but prevents edit wars (as in https://en.wikipedia.org/wiki/Wikipedia:Edit_warring )

Reply:
Yeah I've noticed a ton of redundant parentheses around, and IMO it makes it generally makes it harder to read. What is this, lisp?

FWIW specifically for `OOT_DEBUG && (REG(1) == 0)` vs `OOT_DEBUG && (REG(1) == 0)`, the `OOT_DEBUG && (REG(1) == 0)` seems to be winning

Reply:
I agree and would prefer without but again edit wars lol

Reply:
Fine, I've added redundant parentheses around all these in this file